draft-ietf-quic-manageability.md

```diff
@@ -493,18 +493,23 @@ carried on the wire) unless noted.
 The QUIC wire image is not specifically designed to be distinguishable from
 other UDP traffic.
 
-The only application binding currently defined for QUIC is HTTP
-{{?QUIC-HTTP}}. HTTP over QUIC uses UDP port 443 by default,
+The only application binding defined by the IETF QUIC WG is HTTP/3
+{{?QUIC-HTTP}} at the time of this writing; however, work to run other
+applications over QUIC is ongoing, so an assumption that all QUIC traffic
```

Comment:
Not only is the specification work ongoing, but other protocols are already being deployed in the wild (and have been for quite a while).